### PR TITLE
Add initial version of the opentelemetry-collector chart

### PR DESCRIPTION
**Pull request by dmitryax**
The change was initially submitted and pre-reviewed in contrib repo: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/1026

### CODE REVIEW

**Review by naseemkullah — APPROVED** [review]

**Comment by naseemkullah** [comment]
Do you plan to first setup the repo as per https://github.com/helm/charts-repo-actions-demo ?

**Comment by dmitryax** [comment]
@naseemkullah yes, as the following PR

**Comment by dmitryax** [comment]
for lint-test: https://github.com/open-telemetry/opentelemetry-helm-charts/pull/5

**Comment by dmitryax** [comment]
and release action as a separate PR: https://github.com/open-telemetry/opentelemetry-helm-charts/pull/6

**Review by naseemkullah — APPROVED** [review]

File: charts/opentelemetry-collector/README.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+```yaml
+agentCollector:
+  enabled: false
+standaloneCollector:
+  enabled: true
+```
+
+By default collector has the following receivers enabled:
+- **metrics**: OTLP and prometheus. Prometheus is configured only for scraping collector's own metrics.
+- **traces**: zipkin and jaeger (thrift and grpc).
```
Comment (tigrannajaryan) [inline charts/opentelemetry-collector/README.md:42]: Why don't we also enable OTLP receiver for traces?

File: charts/opentelemetry-collector/templates/config/_metrics.tpl [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        scrape_interval: 10s
+        static_configs:
+        - targets: ["${MY_POD_IP}:8888"]
+exporters:
+  {{ .Values.telemetry.metrics.exporter.type }}:
+    {{- .Values.telemetry.metrics.exporter.config | toYaml | nindent 4 }}
+service:
+  pipelines:
+    metrics:
+      receivers: [prometheus]
```
Comment (tigrannajaryan) [inline charts/opentelemetry-collector/templates/config/_metrics.tpl:20]: otlp receiver doesn't seem to be enabled but README says it should.

**Review by tigrannajaryan — COMMENTED** [review]
This is only traces and metric, right? No logs support yet.

File: charts/opentelemetry-collector/templates/config/_metrics.tpl (hunk 2 above)
Comment (dmitryax) [reply charts/opentelemetry-collector/templates/config/_metrics.tpl]: thank for catching it, missed it. fixed now

File: charts/opentelemetry-collector/README.md (hunk 1 above)
Comment (dmitryax) [reply charts/opentelemetry-collector/README.md]: it is enabled, missed to mention it in the docs. fixed

**Comment by dmitryax** [comment]
> This is only traces and metric, right? No logs support yet.
I didn't include logs by default, but user car set it up

**Review by tigrannajaryan — APPROVED** [review]

**Pull request merged**